<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreditCalc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icon/icon.qrc">
    <normaloff>:/Credit-Calculator.png</normaloff>:/Credit-Calculator.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 94, 94);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="amountCreditLayout">
     <item>
      <widget class="QLabel" name="amountCreditLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Общая сумма кредита</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="amountCreditDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="suffix">
        <string> руб.</string>
       </property>
       <property name="minimum">
        <double>1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000001463069523067487303097004298786465505927861078716979636425114821591040.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="termCreditLayout">
     <item>
      <widget class="QLabel" name="termCreditLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Срок кредита</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="termCreditInputLayout">
       <item>
        <widget class="QSpinBox" name="termCreditSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="termCreditRadioButtonYear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
         </property>
         <property name="text">
          <string>Лет</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">termCredit</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="termCreditRadioButtonMonth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
         </property>
         <property name="text">
          <string>Месяцев</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">termCredit</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="procentageCreditLayout">
     <item>
      <widget class="QLabel" name="procentageCreditlabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Процентная ставка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="procentageCreditDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
       <property name="value">
        <double>9.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="monthPaymentLayout">
     <item>
      <widget class="QLabel" name="monthPaymentLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Ежемесячный платеж</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="monthPaymentLabelOutput">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="overpaymentLayout">
     <item>
      <widget class="QLabel" name="overpaymentLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Переплата по кредиту</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="overpaymentLabelOutput">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="AllPaymentsLayout">
     <item>
      <widget class="QLabel" name="AllPaymentLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>Общая сумма выплат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AllPaymentsLabelOutput">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="resultLayout">
     <item>
      <widget class="QPushButton" name="diffButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
	border-radius: 15px;
}
QPushButton:pressed  {
color: rgb(255, 255, 255);
	background-color: rgb(52, 52, 52);
	border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string>Посчитать дифференцированный кредит</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AnnButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
	border-radius: 15px;
}
QPushButton:pressed  {
color: rgb(255, 255, 255);
	background-color: rgb(52, 52, 52);
	border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string>Посчитать аннуитетный кредит</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon/icon.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="termCredit"/>
 </buttongroups>
</ui>
